types/webpack-dev-server/index.d.ts

```diff
@@ -13,7 +13,11 @@ declare namespace WebpackDevServer {
     interface Configuration {
         contentBase?: string;
         hot?: boolean;
-        https?: boolean;
+        https?: boolean | {
+            key: string,
```

Comment:
Doesn't the following line imply that key and cert are optional (i.e. will use a fake cert)
https://github.com/webpack/webpack-dev-server/blob/97484a9cebad152e67b2fa5ca40058962295675c/lib/Server.js#L463

Reply:
Also can you add the rest of the params?
https://github.com/webpack/webpack-dev-server/blob/97484a9cebad152e67b2fa5ca40058962295675c/lib/Server.js#L375